### ISSUE TITLE
Ability to replace alert left image with customized component

### ISSUE THREAD

**Issue by jack2684** [testshallpass/react-native-dropdownalert#106]
#### Issue Description

Want to have more flexible functionality for the drop message, and do some customized stuff.  Essentially, give a props such that an user and replace this line of with their own component:

https://github.com/testshallpass/react-native-dropdownalert/blob/53fa16d6874832081b8a6bdfc6fd8403df1631dd/DropdownAlert.js#L431

#### Expected Results

Adding a `dropAlertLeft` props, could be something like this (in this case, `iamgeSrc` props is not effective):
```javascript
<DropdownAlert
  ref={(ref) => this.dropdown = ref}
  closeInterval={1500}
  dropAlertLeft={() => <Component> My customized component </Component>}
/>
```

**Comment by Aleksandar-FFWD** [comment]
I would like to see this soon as well. Would be nice to replace it with a `svg icon` that supports `react-native-vector-icons`

**Comment by testshallpass** [comment]
Close #110 merged and released v3.3.0

**Comment by Aleksandar-FFWD** [comment]
@testshallpass Thanks ! 👍
